modules/osdk-ensuring-operator-workloads-run-restricted-psa.adoc

```diff
@@ -0,0 +1,39 @@
+// Module included in the following assemblies:
+//
+// * operators/operator_sdk/osdk-complying-with-psa.adoc
+
+:_content-type: PROCEDURE
+[id="osdk-ensuring-operator-workloads-run-restricted-psa_{context}"]
+= Ensuring Operator workloads run in namespaces set to the restricted pod security level
+
+If you ensure your Operator workloads can run in namespaces set to the `restricted` pod security level, your Operator project can run on a wide variety of deployments and environments. 
+
+[WARNING]
+====
+* You must leave the `runAsUser` field empty. If your image requires a user, ensure the `userID` is defined in the image and not by the security context.
+* Use a workload resource, such as `Deployment` or `StatefulSet`, to manage your Operator's pod. If your Operator creates workload pods by using an account with escalated SCC privileges, the pods might fail.
```

Comment:
(the reality is most operators are probably not creating pods directly anyway, so it's debatable how much we should get into this, and where.  I think it would be best to ensure we have really good documentation about how the PSA label syncer works, elsewhere in the documentation, including the implications on direct pod creation, and then refer people to that)

Reply:
Addressed in 52d26a1

Reply:
I created OSDOCS-5757 to track the follow up work. It is only a stub right now, but we can connect to write a better description and follow up tasks.

Reply:
I removed the second warning. Please let me know if you think that is the right approach. As a rule of thumb, we are told that our downstream docs should attempt to cover 80% of use cases. I am not sure if that heuristic is helpful in this instance.

Reply:
i think that's fine